tastyworks/models/option.py

```diff
@@ -6,21 +6,40 @@
 
 from tastyworks.models.security import Security
 from tastyworks.models.underlying import UnderlyingType
+from tastyworks.models.greeks import Greeks
 
 
 class OptionType(Enum):
     PUT = 'P'
     CALL = 'C'
 
 
+class ExpirationType(Enum):
+    WEEKLY = 'Weekly'
+    REGULAR = 'Regular'
+    QUARTERLY = 'Quarterly'
+
+
+class SettlementType(Enum):
+    AM = 'AM'
+    PM = 'PM'
+
+
 @dataclass
 class Option(Security):
     ticker: str
-    expiry: date
+    expiration_date: date
     strike: Decimal
     option_type: OptionType
     underlying_type: UnderlyingType
+    # quote: Quote
+    dte: int = 0
     quantity: int = 1
+    symbol: str = ''
+    expiration_type: str = None
```

Comment:
Should this be of type `ExpirationType`?

Reply:
Does this type already exist or does it need to be created?